src/main.zig

```diff
@@ -215,11 +215,11 @@ pub const std_options: std.Options = .{ // MARK: std_options
 
 fn initLogging() void {
 	logFile = null;
-	std.fs.cwd().makePath("logs") catch |err| {
+	files.cwd().makePath("logs") catch |err| {
 		std.log.err("Couldn't create logs folder: {s}", .{@errorName(err)});
 		return;
 	};
-	logFile = std.fs.cwd().createFile("logs/latest.log", .{}) catch |err| {
+	logFile = files.cwd().createFile("logs/latest.log") catch |err| {
```

Comment:
Since logging is kind of a special case, I think it would be fine to keep using `std.fs` here, instead of adding this extra function to `main.files`.